addons/web/static/src/core/dropdown/dropdown_navigation_hook.js

```diff
@@ -70,7 +75,9 @@ export function useDropdownNavigation() {
     const menuRef = useRef("menuRef");
     /** @type {MenuElement[]} */
     let menuElements = [];
-    useEffect(() => {
+
+    let cleanupDropdownNavigation;
+    const setupDropdownNavigation = () => {
```

Comment:
Please rename to something like:
```suggestion
    let cleanupMenuElements;
    const refreshMenuElements = () => {
```